src/webgl/p5.RendererGL.js

```diff
@@ -478,26 +488,36 @@ p5.RendererGL.prototype.rotateZ = function(rad) {
  * @return {[type]} [description]
  */
 p5.RendererGL.prototype.push = function() {
-  uMVMatrixStack.push(this.uMVMatrix.copy());
+  uMMatrixStack.push(this.uMMatrix.copy());
 };
 
 /**
  * [pop description]
  * @return {[type]} [description]
  */
 p5.RendererGL.prototype.pop = function() {
-  if (uMVMatrixStack.length === 0) {
+  if (uMMatrixStack.length === 0) {
     throw new Error('Invalid popMatrix!');
   }
-  this.uMVMatrix = uMVMatrixStack.pop();
+  this.uMMatrix = uMMatrixStack.pop();
 };
 
 p5.RendererGL.prototype.resetMatrix = function() {
-  this.uMVMatrix = p5.Matrix.identity();
+  this.uMMatrix = p5.Matrix.identity();
+  this.uVMatrix = p5.Matrix.identity();
   this.translate(0, 0, -800);
   return this;
 };
 
+p5.RendererGL.prototype.cameraTranslate = function(x, y, z) {
```

Comment:
perhaps we can adopt an approach similar to: https://processing.org/reference/beginCamera_.html where translations and rotations are simply applied to a matrix in our MVMatrix stack?